pkg/apiserver/handlers.go

```diff
@@ -73,13 +89,17 @@ func RateLimit(rl util.RateLimiter, handler http.Handler) http.Handler {
 			handler.ServeHTTP(w, req)
 			return
 		}
-		// Return a 429 status indicating "Too Many Requests"
-		w.Header().Set("Retry-After", "1")
-		w.WriteHeader(errors.StatusTooManyRequests)
-		fmt.Fprintf(w, "Rate limit is 10 QPS or a burst of 200")
+		tooManyRequests(w)
 	})
 }
 
+func tooManyRequests(w http.ResponseWriter) {
+	// Return a 429 status indicating "Too Many Requests"
+	w.Header().Set("Retry-After", "1")
+	w.WriteHeader(errors.StatusTooManyRequests)
+	fmt.Fprintf(w, "Rate limit is 10 QPS or a burst of 200")
+}
```

Comment:
Constantized the RetryAfter.
Changed the error message, since it wasn't accurate anyway.